end-to-end-test/remote/specs/core/groupComparison.spec.js

```diff
@@ -1,126 +1,175 @@
-            browser.pause(100);
-            assert.equal($(SampleCreateGroupButton).isEnabled(), true);
-            assert.equal($(PatientCreateGroupButton).isEnabled(), false);
+        it('select from sample venn diagram', async () => {
+            await jsApiClick('rect[data-test="sample0,1VennRegion"]');
+            await browser.pause(100);
+            assert.equal(
+                await (await getElement(SampleCreateGroupButton)).isEnabled(),
+                true
+            );
+            assert.equal(
+                await (await getElement(PatientCreateGroupButton)).isEnabled(),
+                false
+            );
         });
 
-        it('click sample venn diagram create group button', function() {
-            $(SampleCreateGroupButton).click();
-            $('div.rc-tooltip-inner').waitForExist({ timeout: 20000 });
-            browser.pause(100);
+        it('click sample venn diagram create group button', async () => {
+            await (await getElement(SampleCreateGroupButton)).click();
+            await getElement('div.rc-tooltip-inner', { timeout: 20000 });
+            await browser.pause(100);
             assert.equal(
-                $('[data-test="sampleGroupNameInputField"]').isDisplayed(),
+                await (
+                    await getElementByTestHandle('sampleGroupNameInputField')
+                ).isDisplayed(),
                 true,
                 'group name input exists'
             );
             assert.equal(
-                $('[data-test="sampleGroupNameSubmitButton"]').isEnabled(),
+                await (
+                    await getElementByTestHandle('sampleGroupNameSubmitButton')
+                ).isEnabled(),
                 false
             );
         });
 
-        it('sample venn diagram: group name exists, should disable submit button', function() {
-            $('[data-test="sampleGroupNameInputField"]').setValue(
-                'GARS mutant'
-            );
-            browser.pause(100);
-            $('[data-test="sampleDuplicateGroupNameMessage"]').waitForExist({
+        it('sample venn diagram: group name exists, should disable submit button', async () => {
+            await (
+                await getElementByTestHandle('sampleGroupNameInputField')
+            ).setValue('GARS mutant');
+            await browser.pause(100);
+            await getElementByTestHandle('sampleDuplicateGroupNameMessage', {
                 timeout: 20000,
             });
             assert.equal(
-                $('[data-test="sampleDuplicateGroupNameMessage"]').getText(),
+                await (
+                    await getElementByTestHandle(
+                        'sampleDuplicateGroupNameMessage'
+                    )
+                ).getText(),
                 'Another group already has this name.'
             );
             assert.equal(
-                $('[data-test="sampleGroupNameSubmitButton"]').isEnabled(),
+                await (
```

Comment:
i wish there was some way we could chain this but i think there isn't any way.  (i mean the await await thing)